packages/cli/README.md

```diff
@@ -97,7 +109,7 @@ USAGE
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
```

Comment:
I don't know what did:create does.  This might actually use --key but would be good to confirm